config/config.yaml

```diff
@@ -9,38 +9,38 @@ kraken:
     litmus_uninstall: False                                # If you want to uninstall litmus if failure
     chaos_scenarios:                                       # List of policies/chaos scenarios to load
         -   container_scenarios:                                 # List of chaos pod scenarios to load
-            - -    scenarios/container_etcd.yml
+            - -  scenarios/container_etcd.yml
         -   pod_scenarios:
-            - -    scenarios/etcd.yml
-            - -    scenarios/regex_openshift_pod_kill.yml
-              -    scenarios/post_action_regex.py
+            - -    https://raw.githubusercontent.com/cloud-bulldozer/kraken/master/scenarios/etcd.yml
```

Comment:
Lets update these links to have the new organization and branch 

Ex.) https://raw.githubusercontent.com/chaos-kubox/krkn/main/scenarios/etcd.yml